Formula/go.rb

```diff
@@ -47,11 +37,11 @@ class Go < Formula
   depends_on :macos => :mountain_lion
 
   # Should use the last stable binary release to bootstrap.
-  # More explicitly, leave this at 1.7 when 1.7.1 is released.
+  # More explicitly, leave this at 1.8 when 1.8.1 is released.
```

Comment:
It's just a tool we've used in the past for easier build failure debugging, but upstream themselves the last time I checked used Go 1.4 to bootstrap. We moved off of that system because for a while the 1.4 branch died on Sierra, which may or may not still be the case.